fix: adding a fake request to execute in async servers

DeimerM opened the pull request:
Hi team!

This PR is needed to implement programs in a stratus installation, you can read more about this in the following link [more context](https://3.basecamp.com/3966315/buckets/12732851/documents/5430547746), additionally, this is the solution for the following issue #34.

## How to test it.

1. Create a local environment with mango, if you want to see the error you have to use the current version of eox-hooks.
2. Configure your programs, to configure programs. you also have to configure grading and certificates, this is mandatory because this functionality needs that a trigger will be generated when you request a certificate, if you want to know further information about how to configure programs, these links are very useful.
   -  [Multi Courses Programs Public Documenation](https://docs.edunext.co/en/public/external/prepare_test_launch_courses/multi-course-programs.html)
   -  [Multi Courses Programs Internal Documentation](https://docs.edunext.co/en/latest/internal/special-features/programs/programs.html)
3. if you have your programs working, then, you can see the logs on the lms or lms-worker container, it's important to configure your local environment for working with workers.
4. when you see the error, further information about the error is in this issue #34, if you want to solve it, you can install eox-hooks with the following branch `dmh/fixing-programs` to test that the issue is fixed.

mariajgrimaldi commented on eox_hooks/actions.py:
```diff
@@ -20,6 +21,28 @@
 log = logging.getLogger(__name__)
 
 
+class FakeRequest:
+    """
+    This class represents a fake request, this is needed when the action
+    "trigger_grades_assignment" is executed in an async server.
+    """
+    META = {}
+
+    @staticmethod
+    def is_secure():
+        """
+        Returns a True to mock-up https.
+        """
+        return True
+
+    @staticmethod
+    def get_host():
+        """
+        Returns a string with the site name to mock-up the host.
+        """
+        return settings.SITE_NAME
```
This looks great! can you move it to utils? it might make more sense there. What do you think?

DeimerM replied:
yes, you are right, this change is done!! thanks a lot.

MaferMazu commented:
@DeimerM, Can you add how to test it?

DeimerM commented:
> @DeimerM, Can you add how to test it and correct the commit message?

Hi @MaferMazu, sorry for my delayed response, I was a little bit busy, but I have updated the comment with some useful steps to test it, please let me know if these steps are useful or if I have to tweak it a little more, thanks a lot for your help.

MaferMazu commented:
@DeimerM Thanks for the explanation. For me looks good, but I suggest that it's better if you rename the commit, right now is `docs(bumpversion): v2.0.1 → 3.0.0`; please rebase your branch and change the commit message to something like your PR name, and for me looks good.

What do you think, @Alec4r @Henrrypg?

DeimerM commented:
> @DeimerM Thanks for the explanation. For me looks good, but I suggest that it's better if you rename the commit, right now is `docs(bumpversion): v2.0.1 → 3.0.0`; please rebase your branch and change the commit message to something like your PR name, and for me looks good.
> 
> What do you think, @Alec4r @Henrrypg?

Hi @MaferMazu , thanks a lot for your help I have renamed the commit and rebased the branch with master. :)

MaferMazu reviewed (approved):
Looks good to me.

What do you think, @Alec4r @Henrrypg, we can merge this?

JuanDavidBuitrago commented:
Hi @DeimerM, you need to continue working on it, or what do you need to close this PR?

DeimerM commented:
> Hi @DeimerM, you need to continue working on it, or what do you need to close this PR?

Hi @JuanDavidBuitrago Can I merge it ? thanks a lot for your help
